### PR TITLE
fix messageLinkEmbeds, moreUserTags

### PR DESCRIPTION
**Pull request by sunnniee**
still a draft
- waiting for SettingsStoreAPI fix/replacement for MessageLinkEmbeds
- user tags don't show in chat except for webhooks, the patch works, i do not know what is wrong and i am going insane

### CODE REVIEW

**Comment by sunnniee** [comment]
ready

File: src/plugins/messageLinkEmbeds/index.tsx [hunk 1]
```diff
@@ -46,11 +45,12 @@ const messageCache = new Map<string, {
...
 
 const Embed = LazyComponent(() => findByCode(".inlineMediaEmbed"));
-const ChannelMessage = LazyComponent(() => find(m => m.type?.toString()?.includes('["message","compact","className",')));
+const AutoModEmbed = LazyComponent(() => findByCode(".withFooter]:", "childrenMessageContent:"));
+const ChannelMessage = LazyComponent(() => find(m => m.type?.toString()?.includes("renderSimpleAccessories)")));
 
 const SearchResultClasses = findByPropsLazy("message", "searchResult");
 
-let AutoModEmbed: React.ComponentType<any> = () => null;
+const SettingsStores = findByPropsLazy("MessageDisplayCompact");
```
Comment (Vendicated) [inline src/plugins/messageLinkEmbeds/index.tsx:53]:
```suggestion
const TextAndImageSettingsStores = findByPropsLazy("MessageDisplayCompact");
```
and maybe move to webpack/commons/settingsStores.ts?

File: src/plugins/moreUserTags/index.tsx [hunk 2]
```diff
@@ -251,11 +247,16 @@ export default definePlugin({
         },
         // in profiles
         {
-            find: "showStreamerModeTooltip:",
-            replacement: {
+            find: ",overrideDiscriminator:",
+            replacement: [{
```
Comment (Vendicated) [inline src/plugins/moreUserTags/index.tsx:251]: nit: put these objects on new lines for consistency with the rest of the plugins

File: src/plugins/moreUserTags/index.tsx [hunk 3]
```diff
@@ -248,15 +248,17 @@ export default definePlugin({
...
-                // it's either this or extremely long lookbehind
-                match: /user:\i,nick:\i,/,
-                replace: "$&moreTags_channelId,"
-            }, {
-                match: /,botType:(\i\((\i)\)),/g,
-                replace: ",botType:$self.getTag({user:$2,channelId:moreTags_channelId,origType:$1,location:'not-chat'}),"
-            }]
+            replacement: [
+                {
+                    // prevent channel id from getting ghosted
```
Comment (Vendicated) [inline src/plugins/moreUserTags/index.tsx:253]: shadowed is what this is suppopsed to mean?

**Pull request merged**